### DISCUSSION TITLE
Scan QR camera permission issue

### DISCUSSION THREAD

**Discussion by kjincgroup** [BitCannaGlobal/WeedMobile#86] [General]
If I denied for the first time, it will not ask for granting allow camera access permission again, instead, it shows a message "user denied camera access permission"

I expect it will ask for allowing  camera access permission and does not require user granting the permission in Settings app manually.

![scan_qr](https://github.com/BitCannaGlobal/WeedMobile/assets/111372349/71de0df4-9446-4b13-8fc3-d2cf44f070e3)

**Comment by alibabaedge** [comment]
But it's a usual thing, when you already denied the permission. If you want asking every time, you probably should choose "permit access for this time". If I'm not mistaken, in iOS it works something like that)

**Comment by RaulBernal** [comment]
> If I denied for the first time, it will not ask for granting allow camera access permission again, instead, it shows a message "user denied camera access permission"
> 
> I expect it will ask for allowing camera access permission and does not require user granting the permission in Settings app manually.
> 
> ![scan_qr](https://private-user-images.githubusercontent.com/111372349/305684533-71de0df4-9446-4b13-8fc3-d2cf44f070e3.jpeg?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MDkyOTU5NjQsIm5iZiI6MTcwOTI5NTY2NCwicGF0aCI6Ii8xMTEzNzIzNDkvMzA1Njg0NTMzLTcxZGUwZGY0LTk0NDYtNGIxMy04ZmMzLWQyY2Y0NGYwNzBlMy5qcGVnP1gtQW16LUFsZ29yaXRobT1BV1M0LUhNQUMtU0hBMjU2JlgtQW16LUNyZWRlbnRpYWw9QUtJQVZDT0RZTFNBNTNQUUs0WkElMkYyMDI0MDMwMSUyRnVzLWVhc3QtMSUyRnMzJTJGYXdzNF9yZXF1ZXN0JlgtQW16LURhdGU9MjAyNDAzMDFUMTIyMTA0WiZYLUFtei1FeHBpcmVzPTMwMCZYLUFtei1TaWduYXR1cmU9ZjhiN2U3NGJmYmYzYmJkNzA3YTk3ZmU1MDEzYmUxMmMxMTM3ZWJhMjg2ZjAzNjNhMTdhMDE4NDMwZDE0MTJlNCZYLUFtei1TaWduZWRIZWFkZXJzPWhvc3QmYWN0b3JfaWQ9MCZrZXlfaWQ9MCZyZXBvX2lkPTAifQ.P4x6kVVcnjcMLfncQoqMm7wMnJq_ttzwPdjKbVyZZbw)

This capture is at iOS or Android?
What device?
Thanks

**Comment by RaulBernal** [comment]
At this moment iOS release requires a manual enabling at iOS options.
The SDK which face it need to be improved. 

So at this moment is a "normal" or "expected" behavior for iOS. 
Tested in Android and is OK.

**Comment by kjincgroup** [comment]
Understood. Thank you
